### DISCUSSION TITLE
send local gif

### DISCUSSION THREAD

**Discussion by vipguy** [tomer8007/kik-bot-api-unofficial#250] [Q&A]
is it possible to send local gif to group chat ?
if message == "gifs":
            # Extract the path to the GIF file
            gif_file_path = "var\\giftest.gif"
            # Send the GIF file to the user
            self.client.send_gif_image(chat_message.from_jid, gif_file_path)

**Comment by tomer8007** [comment]
It's not possible right now. You can only send GIFs from tendor.com

**Comment by vipguy** [comment]
Ok ty
